builtin/logical/pki/path_roles.go

```diff
@@ -187,7 +187,14 @@ this value in certificates issued by this role.`,
 				Default: false,
 				Description: `
 If set, certificates issued/signed against this role will have Vault leases
-attached to them. Defaults to "false".`,
+attached to them. Defaults to "false".
+
+For certificates associated with leases, when the leases expire, Vault will add
```

Comment:
This isn't true -- when it expires it won't go on the CRL because the certificate has expired too. It will only go in the CRL if you revoke it using the lease ID.